pdc/apps/common/viewsets.py

```diff
@@ -200,7 +200,8 @@ def initial(self, request, *args, **kwargs):
             self._check_ordering_keys(request)
 
     def _check_ordering_keys(self, request):
-        ordering_keys = request.query_params.get('ordering')
+        # Allow ordering to be defined multiple times and also as comma separated list.
+        ordering_keys = ','.join(request.query_params.getlist('ordering'))
```

Comment:
Yes, currently our server just can use ordering=a,b,c.

I have googled about the definitive standard of the parameter, seems there is no such a standard.
And from my point of view, I think both are okay, but I worry about that the current pdc users may have used the ","  for  a while, if we change it, it will make them confuse.
@hluk What's your thinking?